examples/awssig.rs

```diff
@@ -230,7 +230,7 @@ extern "C" fn ngx_http_awssigv4_commands_set_s3_bucket(
         conf.s3_bucket = (*args.add(1)).to_string();
         if conf.s3_bucket.len() == 1 {
             println!("Validation failed");
-            return ngx::core::NGX_CONF_ERROR as _;
+            return ngx::core::NGX_CONF_ERROR;
```

Comment:
```
fix: correct return type for nginx configuration handlers

Change NGX_CONF_ERROR to *mut c_char and use wrapping_offset() to preserve 
pointer provenance instead of direct cast from isize.
Add NGX_CONF_OK constant for completeness.
```

maybe this commit log is a bit clearer at describing the intent